internal/pointer/pointer.go

```diff
@@ -44,3 +44,11 @@ func GetNonZeroValue[T constraints.Integer](val T) *T {
 
 	return nil
 }
+
+func IntToInt64(source *int) *int64 {
```

Comment:
can you move this closer to the kube package, these methods are a bit of  a hack for converting between the SDK and the operator internal and not really helpful for them to be exposed to all the code base

Reply:
Fixed. Moved to `.../kubernetes/operator/convert/` package

Reply:
thanks